tests/test_schema.py

```diff
@@ -28,46 +28,65 @@
 
 from lsst.utils import getPackageDir
 
-butler = Butler(f'{os.environ["CI_IMSIM_DIR"]}/DATA', collections=['LSSTCam-imSim/runs/ci_imsim'])
-skymap = list(butler.registry.queryDatasets(datasetType='objectTable_tract'))[0].dataId['skymap']
+butler = Butler(
+    f'{os.environ["CI_IMSIM_DIR"]}/DATA', collections=["LSSTCam-imSim/runs/ci_imsim"]
+)
+skymap = list(butler.registry.queryDatasets(datasetType="objectTable_tract"))[0].dataId[
+    "skymap"
+]
 
 
 class TestSchemaMatch(lsst.utils.tests.TestCase):
     """Check the schema of the parquet outputs match the DDL in sdm_schemas"""
 
     def setUp(self):
-        self.butler = Butler(os.path.join(getPackageDir("ci_imsim"), "DATA"),
-                             writeable=False, collections=["LSSTCam-imSim/runs/ci_imsim"])
-        schemaFile = os.path.join(getPackageDir("sdm_schemas"), 'yml', 'imsim.yaml')
+        self.butler = Butler(
+            os.path.join(getPackageDir("ci_imsim"), "DATA"),
+            writeable=False,
+            collections=["LSSTCam-imSim/runs/ci_imsim"],
+        )
+        schemaFile = os.path.join(getPackageDir("sdm_schemas"), "yml", "imsim.yaml")
         with open(schemaFile, "r") as f:
-            self.schema = yaml.safe_load(f)['tables']
+            self.schema = yaml.safe_load(f)["tables"]
 
     def _validateSchema(self, dataset, dataId, tableName):
         """Check column name and data type match between dataset and DDL"""
         info = f"dataset={dataset} tableName={tableName} dataId={dataId}"
 
-        sdmSchema = [table for table in self.schema if table['name'] == tableName]
+        sdmSchema = [table for table in self.schema if table["name"] == tableName]
         self.assertEqual(len(sdmSchema), 1)
-        expectedColumns = {column['name']: column['datatype']
-                           for column in sdmSchema[0]['columns']}
+        expectedColumns = {
+            column["name"]: column["datatype"] for column in sdmSchema[0]["columns"]
+        }
 
-        df = self.butler.get(dataset, dataId)
+        df = self.butler.get(dataset, dataId, storageClass="DataFrame")
```

Comment:
This should be unnecessary after DM-41962.  I would recommend removing this line and testing with that ticket branch.